src/dios-egraphs/src/main.rs

```diff
@@ -55,7 +49,8 @@ fn main() {
 
     // Run rewriter
     eprintln!(
-        "Running egg with timeout {:?}s, width: {:?}",
+        "Running egg on {:?} with timeout {:?}s, width: {:?}",
+        &prog,
```

Comment:
This could be very large for some of our "traditional" Diospyros benchmarks, I think, so it could actually be a little unwieldy to print out. Maybe the LLVM pass wants to print this out itself instead of doing it here?

Reply:
this only prints during diospyros tests now